library/src/com/google/maps/android/geojson/GeoJsonPoint.java

```diff
@@ -0,0 +1,51 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.LatLng;
+
+/**
+ * A GeoJsonPoint geometry contains a single {@link com.google.android.gms.maps.model.LatLng}.
```

Comment:
Slight nit: make a note that this class does not support altitude or any further values. Technically GeoJson allows a point or position to be an unbounded list of values representing "lat, lng, altitude, undefined, ..."